packages/cli-kit/bin/fetch-notifications.js

```diff
@@ -0,0 +1,21 @@
+import fetch from 'node-fetch'
+import {notificationsUrl} from '../dist/public/node/notifications-system.js'
+import {cacheStore} from '../dist/private/node/conf-store.js'
+
+async function fetchNotifications() {
+  try {
+    const url = notificationsUrl()
+    const response = await fetch(url, {signal: AbortSignal.timeout(3 * 1000)})
+    if (response.status === 200) {
+      cacheStore(`notifications-${url}`, await response.text())
+      console.log(`Notifications from ${url} cached successfully`)
+    }
+  } catch (error) {
+    const errorMessage = `Error fetching notifications: ${error.message}`
+    console.error(errorMessage)
+    const {sendErrorToBugsnag} = await import('../dist/public/node/error-handler.js')
+    await sendErrorToBugsnag(errorMessage, 'unexpected_error')
```

Comment:
Ensured that this works from here: https://app.bugsnag.com/shopify/cli/errors/675049cdac655414327f515f?event_id=675049cd010d594612b60000&i=sk&m=cm